admin/marc_order_accounts.pl

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/perl
+
+# A script that allows the user to create an account and profile for auto-creating orders from imported marc files
+# The script displays account details and allows account creation/editing in the first instance
+# If the "run" operation is passed then the script will run the process of creating orders
+
+# Copyright 2023 PTFS Europe Ltd
+#
+# This file is part of Koha.
+#
+# Koha is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# Koha is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Koha; if not, see <http://www.gnu.org/licenses>.
+
+use Modern::Perl;
+use CGI qw ( -utf8 );
+
+use C4::Context;
+use C4::Auth qw( get_template_and_user );
+use C4::Budgets qw( GetBudgets );
+use C4::Output qw( output_html_with_http_headers );
+use C4::Matcher;
+
+use Koha::UploadedFiles;
+use Koha::ImportBatchProfiles;
+use Koha::MarcOrder;
+use Koha::Acquisition::Booksellers;
+use Koha::MarcOrderAccount;
+use Koha::MarcOrderAccounts;
+
+use Data::Dumper; #DELETE
+
+my $input = CGI->new;
+
+my ( $template, $loggedinuser, $cookie ) = get_template_and_user(
+    {
+        template_name   => "admin/marc_order_accounts.tt",
+        query           => $input,
+        type            => "intranet",
+    }
+);
+
+
+my $fileID = $input->param('uploadedfileid');
+if($fileID) {
+    my $profile = Koha::MarcOrderAccounts->find({ id => 1 }); #Replace with ID once form created
+        # Remove once SFTP set up
+    my $upload = Koha::UploadedFiles->find( $fileID );
+    my $filepath = $upload->full_path;
+    # /var/lib/koha/kohadev/tmp/koha_kohadev_upload
+    my $filename = $upload->filename;
+    # new_bib.marcxml
+    my $args = {
+        filename => $filename,
+        filepath => $filepath,
+        profile => $profile,
+    };
+
+
+    my $sth = Koha::MarcOrder->create_order_lines_from_file($args);
+} #DELETE WHEN FORM CREATED AND CRONJOB ACTIVE
+
+
+
+
+my $crypt = Koha::Encryption->new;
```

Comment:
I assume this was originally here for storing a password in the FTP case?

I don't see it used at the moment.